frontend/components/markup-modal/index.js

```diff
@@ -2,28 +2,13 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import Modal from 'react-modal'
 import styled from 'styled-components'
-import {
```

Comment:
Probably you remember that this code was substantially duplicated into https://github.com/badges/shields/pull/2554/files#diff-559d49006536e9e410b609d2668583ec

I didn't bother to DRY it up because I hoped it wouldn't be around for long: https://github.com/badges/shields/pull/2554/files#r245429994